applications/SolidMechanicsApplication/custom_processes/assign_scalar_variable_to_entities_process.hpp

```diff
@@ -39,11 +39,16 @@ class AssignScalarVariableToEntitiesProcess : public Process
     /// Pointer definition of AssignScalarVariableToEntitiesProcess
     KRATOS_CLASS_POINTER_DEFINITION(AssignScalarVariableToEntitiesProcess);
 
-    enum EntityType { NODES, CONDITIONS, ELEMENTS};
+    enum EntityType { NODES, CONDITIONS, ELEMENTS };
```

Comment:
Please change the enum to enum classes, otherwise this will create conflict with the whole namespace (and NODES, CONDITIONS and ELEMENTS are quite used in Kratos namespace)

Reply:
See https://github.com/KratosMultiphysics/Kratos/blob/c3b3d84aad4f0c4cea11b8021daf03d6d387c7cc/kratos/processes/fast_transfer_between_model_parts_process.h#L87 (and is defined inside the class)